README.rst

```diff
@@ -102,6 +102,19 @@ If you use `mamba <https://mamba.readthedocs.org/>`__::
 
 For other ways to install ``pygmt``, see `full installation instructions <https://www.pygmt.org/latest/install.html>`__.
 
+Getting started
++++++++++++++++
+
+As a starting point you can try the following example::
```

Comment:
```suggestion
As a starting point, you can open a Python console or a Jupyter Notebook, and try 
the following example::
```

Reply:
It's only a minor detail, but what is prefered "Jupyter Notebook" (upper-case) or "Jupter notebook" (lower-case)?
Currently, it seems we are mostly using the lower-case version:
```
$ grep "Jupyter" */*
grep: ci/requirements: Is a directory
grep: doc/_static: Is a directory
grep: doc/_templates: Is a directory
grep: doc/api: Is a directory
doc/changes.md:* Add instructions to install pygmt kernel for Jupyter users ([#2153](https://github.com/GenericMappingTools/pygmt/pull/2153))
doc/changes.md:* Support tab auto-completion in Jupyter ([#1282](https://github.com/GenericMappingTools/pygmt/pull/1282))
doc/changes.md:* Improve Figure.show for displaying previews in Jupyter notebooks and external viewers ([#529](https://github.com/GenericMappingTools/pygmt/pull/529))
doc/contributing.md:must be formatted manually. To play nicely with Jupyter and IPython, **keep docstrings
doc/install.rst:interpreter (e.g., in a Jupyter notebook)::
doc/install.rst:* `IPython <https://ipython.org>`__: For embedding the figures in Jupyter notebooks (recommended).
doc/install.rst:Notes for Jupyter users
doc/install.rst:get a ``ModuleNotFoundError`` when importing pygmt in Jupyter, you may need to
doc/install.rst:After that, you need to restart Jupyter, open your notebook, select the
doc/team.md:C API, supporting rich display in Jupyter notebooks, and integrating with
grep: examples/gallery: Is a directory
grep: examples/get_started: Is a directory
grep: examples/projections: Is a directory
grep: examples/tutorials: Is a directory
pygmt/__init__.py:ecosystem as well as support for rich display in Jupyter notebooks.
grep: pygmt/clib: Is a directory
grep: pygmt/datasets: Is a directory
pygmt/figure.py:# Show figures in Jupyter notebooks if available
pygmt/figure.py:    if get_ipython and "IPKernelApp" in get_ipython.config:  # Jupyter Notebook enabled
pygmt/figure.py:        Inserts the preview in the Jupyter notebook output if available,
pygmt/figure.py:            The image resolution (dots per inch) in Jupyter notebooks.
pygmt/figure.py:            The image width (in pixels) in Jupyter notebooks.
pygmt/figure.py:            - **notebook**: PNG preview [Default in Jupyter notebooks]
pygmt/figure.py:                        "or run the script in a Jupyter notebook."
pygmt/figure.py:        For the Jupyter notebook or IPython Qt console.
pygmt/figure.py:        - **notebook**: PNG preview [Default in Jupyter notebooks]
grep: pygmt/helpers: Is a directory
grep: pygmt/src: Is a directory
grep: pygmt/tests: Is a directory
```

The Jupyter docummentation can be found at https://jupyter-notebook.readthedocs.io/en/stable/notebook.html?highlight=jupyter.

Reply:
It seems the Jupyter documentation also inconsistently use "notebook" and "Notebook". I don't have a strong preference on either, but it seems changing "Notebook" to "notebook" is easier.

Reply:
The lower-case version "Jupyter notebook" is fine with me.